en/docs/tutorials/create-your-first-webhook.md

```diff
@@ -0,0 +1,397 @@
+         | **`variable`**         | `label`         |
+         | **`<add-expression>`** | `payload.label` |
+
+         Here, you are adding a variable named `label` of which the value type is a GitHub label. `payload.label` value expression extracts the GitHub label as the value of the variable.
+   
+    4. Click **Save**.
+   
+4. Once the component extracts the GitHub label of an issue, it needs to check whether the value of the `label` parameter is `bug`. To do this, add an If statement as follows:
+
+    1. Click **+** after the `label` variable statement.
+   
+    2. In the **Add Constructs** pane, click **If**.
+
+    3. In the **If** pane, double-click **`<add-expression>`** and enter the following as the expression.
+      
+        ```
+         label is github:Label && label.name == "bug"
+        ```
+      
+        This condition checks whether the GitHub label derived as the value of the `label` variable is equal to `bug`.
+
+    4. Click **Save**.
+
+5. To send an email when a GitHub issue meets the condition you defined in the previous step, let's add the Choreo SendEmail connector as follows:
+
+     1. Click **+** after the If statement (i.e., the **+** on the `then` flow).
+   
+     2. In the **Add Constructs** pane, click **Connector**. 
+   
+     3. In the **Connectors** pane, find the **Choreo SendEmail** connector, click it, and click **Save** to add it.
+   
+     4. Click the **+** after the **sendEmailEp** connector you added.
+   
+     5. In the **Add Constructs** pane, click **Action**.
+   
+     6. To connect the action you are defining to the Choreo SendEmail connector, click **sendEmailEp**.
+   
+     7. Under **Select an Action**, click **Send Email**.
+   
+     8. In the **Recipient** field, you can enter the email address to which the webhook should send the email notification.
+
+         However, this email address can be a sensitive information item that your clients may not want to expose in the code of the webhook. To address this, you can define the recipient as a configurable variable as follows:
+
+          1. In **`recipient = ""`**, click **`""`** and then click the **Add Configurable** icon.
+       
+             ![Configurables icon](../assets/img/tutorials/webhook/configurables-icon.png)
+
+          2. In the **Add Configurable** pane, double-click **conf** and replace it with `toEmail` Here, you are specifying `toEmail` as the configurable name.
+        
+          3. Leave the default configurable variable type (i.e., `string`) and the expression (i.e., **`?`**) unchanged, and click **Add**.
+          
+              When you add this configurable variable with `?` as the expression, the **toEmail** field will be available when you deploy this webhook, allowing you to enter any email address in string format. This way, you do not have to include the actual value for the **toEmail** field in the code.
+
+          4. Click **Add**.
```

Comment:
Seems like this Should be "Click **Save**."

Reply:
Changed as suggested.